Core/Core/Discussions/DiscussionReplyViewController.swift

```diff
@@ -165,7 +164,7 @@ public class DiscussionReplyViewController: UIViewController, ErrorViewControlle
     public override func viewWillAppear(_ animated: Bool) {
         super.viewWillAppear(animated)
         navigationController?.navigationBar.useModalStyle()
-        keyboard = KeyboardTransitioning(view: view, space: keyboardSpace)
+        keyboard = KeyboardTransitioning(view: view, space: nil)
```

Comment:
The keyboardSpace is necessary in iOS 13, because the safe area does not end up including the keyboard.